modules/nlu/src/backend/tools/stopWords.ts

```diff
@@ -0,0 +1,39 @@
+import { createReadStream } from 'fs'
+import fs from 'fs-extra'
+import path from 'path'
+import readline from 'readline'
+
+fs.exists
+
+const StopWordsByLang: _.Dictionary<string[]> = {}
+
+async function loadStopWords(language: string): Promise<string[]> {
+  const fn = path.join(__dirname, `stopwords/${language}.txt`)
+
+  const langSupported: boolean = await Promise.fromCallback(callback => {
+    fs.exists(fn, callback.bind(this, undefined))
+  })
```

Comment:
`if(await fse.exists(filePath))`

Reply:
didn't know about the promise api thanks